charts/consul/values.yaml

```
@@ -471,6 +471,18 @@ server:
     runAsUser: 100
     fsGroup: 1000
 
+  # The container securityContext for each container in the server pods.  In
+  # addition to the [Pod secuirtyContext](#v-server-securitycontext) this can
```

```suggestion
  # addition to the Pod's SecurityContext this can
```

Linking would be nice but we don't do this elsewhere and might break later.